soca/soca_3dvar.yaml

```diff
@@ -32,7 +33,7 @@ variational:
   - geometry:
       mom6_input_nml: mom_input.nml
       fields metadata: fields_metadata.yaml
-    ninner: 100
+    ninner: 150
     gradient norm reduction: 1e-3
```

Comment:
is it standard to put ninner: 150? I guess it won't take larger iteration to reduce to 1e-3 gradient value.